flight/PiOS/inc/pios_can.h

```diff
@@ -36,8 +36,25 @@
 //! The set of CAN messages
 enum pios_can_messages {
 	PIOS_CAN_GIMBAL = 0,
-	PIOS_CAN_LAST = 1
+	PIOS_CAN_ATTITUDE_ROLL_PITCH = 1,
+	PIOS_CAN_ATTITUDE_YAW = 2,
+	PIOS_CAN_BATTERY_VOLT = 3,
+	PIOS_CAN_BATTERY_CURR = 4,
+	PIOS_CAN_RSSI = 5,
+	PIOS_CAN_ALT = 6,
+	PIOS_CAN_FLIGHTSTATUS = 7,
+	PIOS_CAN_GPS_LATLON = 8,
+	PIOS_CAN_GPS_ALTSPEED = 9,
+	PIOS_CAN_GPS_FIX = 10,
+	PIOS_CAN_GPS_VEL = 11,
+	PIOS_CAN_POS = 12,
+	PIOS_CAN_ALARM = 13,
+	PIOS_CAN_LAST
```

Comment:
Oh wait. I don't think it is needed here, because PIOS_CAN_LAST by definition better always be on the last line :)